napalm_eos/eos.py

```diff
@@ -985,16 +985,20 @@ def get_mac_address_table(self):
     def get_route_to(self, destination='', protocol=''):
         routes = dict()
 
+        if protocol.lower() not in ['', 'bgp', 'connected', 'isis', 'ospf', 'rip', 'static']:
```

Comment:
That would be the most optimal, but for the moment let's just raise ```TypeError```.
And do check only if the protocol is specified, otherwise will return all possible route.s